transactionoutbox-core/src/test/java/com/synaos/transactionoutbox/acceptance/TestMySql5.java

```diff
@@ -1,29 +1,32 @@
-package com.gruelbox.transactionoutbox.acceptance;
+package com.synaos.transactionoutbox.acceptance;
 
-import com.gruelbox.transactionoutbox.Dialect;
-import java.time.Duration;
+import com.synaos.transactionoutbox.Dialect;
+import org.junit.jupiter.api.Disabled;
 import org.testcontainers.containers.JdbcDatabaseContainer;
 import org.testcontainers.containers.MySQLContainer;
 import org.testcontainers.junit.jupiter.Container;
 import org.testcontainers.junit.jupiter.Testcontainers;
 
+import java.time.Duration;
+
 @SuppressWarnings("WeakerAccess")
 @Testcontainers
+@Disabled //todo fixme
```

Comment:
Disabled? and todo?

Reply:
We are not able to support mysql 5 with our changes ... so the tests would fail